README.md

```diff
@@ -148,13 +149,22 @@ You can populate below template according to your requirements and use it as you
 
 # BACKUP_CRON_EXPRESSION="0 2 * * *"
 
-# The name of the backup file including the `.tar.gz` extension.
+# The compression algorithm used in conjunction with tar.
+# Valid options are: "gzip" and "zstd".
+# Note that the selection affects the file extension.
+
+# BACKUP_COMPRESSION="gzip"
```

Comment:
I'm wondering if the options here should we `gz` and `zst` instead, so we don't need to map the name to the extension and so forth.

Reply:
I was a bit unsure about this since gzip and zstd are (imo) the proper names but I see some advantage using the shorter variant. Ultimately I changed it to what you asked for since it's your project :).
Please see what I did in my attempt to implement it. I'm still very new to GO :/